src/Aspire.Hosting.PostgreSQL/PostgresBuilderExtensions.cs

```diff
@@ -114,6 +114,73 @@ public static IResourceBuilder<PgAdminContainerResource> WithHostPort(this IReso
         });
     }
 
+    /// <summary>
+    /// Configures the host port that the pgweb resource is exposed on instead of using randomly assigned port.
+    /// </summary>
+    /// <param name="builder">The resource builder for pgweb.</param>
+    /// <param name="port">The port to bind on the host. If <see langword="null"/> is used random port will be assigned.</param>
+    /// <returns>The resource builder for pgweb.</returns>
+    public static IResourceBuilder<PgWebContainerResource> WithHostPort(this IResourceBuilder<PgWebContainerResource> builder, int? port)
+    {
+        return builder.WithEndpoint("http", endpoint =>
+        {
+            endpoint.Port = port;
+        });
+    }
+
+    /// <summary>
+    /// Adds an administration and development platform for PostgreSQL to the application model using pgweb. This version the package defaults to the 2.3-latest tag of the latest container image
```

Comment:
Can you make this more concise and split out any extended explanation into a `<remarks />` tag.